src/vshaxe/helper/PathHelper.hx

```diff
@@ -46,4 +46,9 @@ class PathHelper {
 		}
 		return path;
 	}
+
+	public static function sanitizeComas(path:String) {
```

Comment:
"Commas", not "Comas".

Reply:
Sorry...
Be happy I've not written in polish :rofl:

Reply:
pushed an update with a corrected and more specific name